### PR TITLE
Add http commander

### PR DESCRIPTION
**Pull request by t-mon**

### CODE REVIEW

**Comment by mzanetti** [comment]
dh_install: usr/lib/arm-linux-gnueabihf/nymea/plugins/libnymea_devicepluginhttpcommander.so exists in debian/tmp but is not installed to anywhere

File: httpcommander/devicepluginhttpcommander.h [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ *  Lesser General Public License for more details.                        *
+ *                                                                         *
+ *  You should have received a copy of the GNU Lesser General Public       *
+ *  License along with this library; If not, see                           *
+ *  <http://www.gnu.org/licenses/>.                                        *
+ *                                                                         *
+ * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+
+#ifndef DEVICEPLUGINDHTTPCOMMANDER_H
+#define DEVICEPLUGINDHTTPCOMMANDER_H
```
Comment (mzanetti) [inline httpcommander/devicepluginhttpcommander.h]: D? :D

File: httpcommander/translations/de_DE.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="de_DE">
+</TS>
```
Comment (mzanetti) [inline httpcommander/translations/de_DE.ts]: file to be delete

File: httpcommander/translations/en_US.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1">
+</TS>
```
Comment (mzanetti) [inline httpcommander/translations/en_US.ts]: file to be deleted

File: debian/guh-plugin-httpcommander.install.in [hunk 4]
```diff
@@ -0,0 +1 @@
+usr/lib/@DEB_HOST_MULTIARCH@/guh/plugins/libguh_devicepluginhttpcommander.so
```
Comment (mzanetti) [inline debian/guh-plugin-httpcommander.install.in]: seems the "guh" in here is the reason for the jenkins build failure

**Comment by t-mon** [comment]
Jenkins, retest this please

**Comment by mzanetti** [comment]
Jenkins, land this please

**Pull request merged**